classes/handler/PKPHandler.php

```diff
@@ -332,7 +331,7 @@ public function authorize($request, &$args, $roleAssignments)
         if (!SessionManager::isDisabled()) {
             // Add user roles in authorized context.
             $user = $request->getUser();
-            if ($user instanceof \PKP\user\User || $request->getRouter() instanceof \PKP\core\APIRouter) {
+            if ($user instanceof \PKP\user\User) {
```

Comment:
@asmecher just flagging this for your review. I'm not sure of the implications here.

Reply:
Bozana and I discussed this and decided it would be better to explicitly state the policies in the handlers. Fewer "special cases" mean easier adaptation e.g. to Laravel from Slim (https://github.com/pkp/pkp-lib/issues/7698); this also leaves the decision up to the handler in as fine-grained a way is it likes, as opposed to the previous proposal that made it a handler-wide decision.